common/src/com/thoughtworks/go/domain/Pipeline.java

```diff
@@ -59,7 +59,7 @@ public Pipeline(String pipelineName, BuildCause buildCause, Stage... stages) {
     }
 
     public File defaultWorkingFolder() {
-        return new File(CruiseConfig.WORKING_BASE_DIR + pipelineName);
+        return new File("" + pipelineName);
```

Comment:
It seems odd to be be calling `defaultWorkingFolder` on a pipeline with no name.